test/membrane_rtmp_plugin/rtmp_source_bin_test.exs

```diff
@@ -215,32 +218,38 @@ defmodule Membrane.RTMP.SourceBin.IntegrationTest do
          port \\ 0,
          use_ssl? \\ false
        ) do
+    parent_process_pid = self()
+
+    new_client_callback = fn client_ref, app, stream_key ->
+      send(parent_process_pid, {:client_ref, client_ref, app, stream_key})
+    end
```

Comment:
This one seems to look as a kind of a pattern we repeat in most use cases - what do you think about making it a default `new_client_callback` implementation?